src/SqlFormatter.php

```diff
@@ -156,15 +143,16 @@ public function format(string $string, string $indentString = '  ') : string
                 // First check if this should be an inline parentheses block
                 // Examples are "NOW()", "COUNT(*)", "int(10)", key(`somecolumn`), DECIMAL(7,2)
                 // Allow up to 3 non-whitespace tokens inside inline parentheses
-                $length = 0;
+                $length    = 0;
+                $v         = '';
```

Comment:
Maybe we can find a more meaningful name for this? What looks weird to me is that this variable seems to be used on line 179, and then… never again?

Reply:
this was just a leftover from debugging, removed